### DISCUSSION TITLE
Resize in the context menu of the taskbar.

### DISCUSSION THREAD

**Discussion by HelderRocket** [moudey/Shell#92] [Ideas]
I find it a very good tool, but I would like to be able to change the size of the taskbar context menu, as it feels really small.
![image](https://user-images.githubusercontent.com/104467221/208441406-b3b2c7e3-1365-4b93-bcb9-da527a522652.png)

**Comment by moudey** [comment]
Add this code to `static` section in `shell.shl` file. Where he makes an amendment to the name whose letters exceed 30 characters.
```
item(where=@(this.title.len > 30) title=@(str.sub(this.title,0,30) + "..."))
```
